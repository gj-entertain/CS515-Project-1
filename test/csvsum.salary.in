Name,Age,Salary
John,30,50000
Jane,25,60000
Doe,40,55000