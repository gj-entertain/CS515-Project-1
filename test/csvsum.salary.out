Sum of column 1: 95.0
Sum of column 2: 165000.0
